cmd/gomd/commands.go

```diff
@@ -137,6 +137,13 @@ func (a * app) cmdSync() error {
 func (a * app) cmdSwitch() error {
 	leftPath := a.left.Folder.Path
 	rightPath := a.right.Folder.Path
-	a.left.Folder.SetDir(rightPath)
-	return a.right.Folder.SetDir(leftPath)
+	err1 := a.left.Folder.SetDir(rightPath)
+	if(err1 != nil){
+		return err1
+	}
+	err2 := a.right.Folder.SetDir(leftPath)
+	if err2 != nil{
+		return err2
+	} 
+	return nil	
```

Comment:
That is not a idiomatic solution. The pattern is:
```
err := a.left.foo()
if err != nil {
  return err
}
return a.right.foo()
```

`err1`, `err2`, ... should not be used. Just reuse the first `err` variable. That value should be `nil` because we check after every time.

Reply:
I made another commit with the changes mentioned above. Thanks for very valuable advice.

Reply:
Yes that is it. Very good.